scripts/download-diff-metadata.ps1

```diff
@@ -49,20 +49,28 @@ if ($branch -ne $env:targetBranch) {
     git pull origin $env:targetBranch --ff-only | Write-Host
 }
 
-# Download the metadata from livesite.
-$url = "https://graph.microsoft.com/{0}/`$metadata" -f $env:endpointVersion
-$metadataFileName = "{0}_metadata.xml" -f $env:endpointVersion
-$pathToLiveMetadata = Join-Path -Path ($pwd).path -ChildPath $metadataFileName
-$client = new-object System.Net.WebClient
-$client.Encoding = [System.Text.Encoding]::UTF8
-Write-Host "Attempting to downloaded metadata from $url to $pathToLiveMetadata" -ForegroundColor DarkGreen
-$client.DownloadFile($url, $pathToLiveMetadata)
-Write-Host "Downloaded metadata from $url to $pathToLiveMetadata" -ForegroundColor DarkGreen
+if ($env:endpointVersion -eq "beta"){
```

Comment:
Rather than evaluating the endpoint version, maybe we should consider having the script simply evaluate if `$env:InputMetadataFile` is a URI or a file path and then use the result to set the `$content`?
That way when we turn this generation source on for v1, we will only need to change the environment variable rather than changing/modifying this script again?